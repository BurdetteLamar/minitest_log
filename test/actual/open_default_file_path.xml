<log>
  <summary verdicts='1' failures='0' errors='0'/>
  <data name='file_path' class='String' size='9'>
    ./log.xml
  </data>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <expected class='Integer' value='0'/>
    <actual class='Integer' value='0'/>
  </verdict>
</log>
