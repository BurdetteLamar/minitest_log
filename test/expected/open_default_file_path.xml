<log>
  <summary verdicts='1' failures='0' errors='0'/>
  <data name='file_path' class='String' size='9'>
    ./log.xml
  </data>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
